ckan/templates/snippets/license.html

```diff
@@ -2,19 +2,18 @@
   {% if 'license_url' in pkg_dict %}
     <a href="{{ pkg_dict.license_url }}" rel="dc:rights">{{ pkg_dict.license_title }}</a>
   {% else %}
-    <span property="dc:rights">{{ pkg_dict.license_title }}</span>
+    {% if pkg_dict.license_id %}
+      <span property="dc:rights">{{ pkg_dict.license_id }}</span>
```

Comment:
As i understand, it will print machine readable name of license. i.e, **cc-by** instead of **Creative Commons Attribution**